tests/Unit/Infrastructure/FileSyncer/PhpFileSyncerTest.php

```diff
@@ -85,44 +101,43 @@ public function testCopy($source, $givenDestination, $expectedDestination, $give
         self::assertSame((string) $expectedTimeout, ini_get('max_execution_time'), 'Correctly set process timeout.');
     }
 
-    public function providerCopyWithOptionalParams(): array
+    public function providerSyncWithOptionalParams(): array
     {
         return [
             [
                 'source' => 'lorem',
                 'givenDestination' => '',
                 'expectedDestination' => '',
                 'givenExclusions' => null,
-                'expectedExclusions' => [''],
+                'expectedExclusions' => [],
                 'callback' => null,
                 'givenTimeout' => null,
                 'expectedTimeout' => 0,
             ],
             [
                 'source' => '',
                 'givenDestination' => 'lorem',
-                'expectedDestination' => 'lorem',
+                'expectedDestination' => 'lorem/',
                 'givenExclusions' => [],
-                'expectedExclusions' => ['lorem'],
+                'expectedExclusions' => [],
                 'callback' => null,
                 'givenTimeout' => null,
                 'expectedTimeout' => 0,
             ],
             [
                 'source' => '',
-                'givenDestination' => $this->fixSeparators('lorem/ipsum/'),
-                'expectedDestination' => $this->fixSeparators('lorem/ipsum'),
+                'givenDestination' => 'lorem/ipsum',
+                'expectedDestination' => 'lorem/ipsum/',
                 'givenExclusions' => [
                     'amet',
                     'consectetur',
-                    $this->fixSeparators('lorem/ipsum'),
-                    $this->fixSeparators('lorem/ipsum'),
-                    $this->fixSeparators('lorem/ipsum'),
+                    'amet',
+                    'consectetur',
```

Comment:
Is this duplication of `consectetur` intentional? If so, maybe a comment would be useful.

Reply:
Yes--to test that exclusions get de-duped. I've added a comment.